docs/api/ref/schemas/product.yaml

```diff
@@ -1,2043 +1,29 @@
-            type: string
-          isAllergenRelevantDataProvided:
-            type: string
-          lastChangeDateTime:
-            type: string
-          partyName:
-            type: string
-          productionVariantDescription:
-            type: string
-          publicationDateTime:
-            type: string
-  states:
-    type: string
-  states_hierarchy:
-    type: array
-    items:
-      type: string
-  states_tags:
-    type: array
-    items:
-      type: string
-  stores:
-    type: string
-    description: |
-      Distributor name.
-    example: Walmart
-  stores_tags:
-    type: array
-    items:
-      type: string
-  teams:
-    type: string
-  teams_tags:
-    type: array
-    items:
-      type: string
-  traces:
-    type: string
-    description: |
-      List of substances that might cause allergies
-      that are present in trace amounts in the product
-      (this does not include the ingredients, as they
-      are not only present in trace amounts).
-      It is taxonomized with the allergens taxonomy.
-  traces_from_ingredients:
-    type: string
-  traces_from_user:
-    type: string
-  traces_hierarchy:
-    type: array
-    items:
-      type: object
-  traces_lc:
-    type: string
-  traces_tags:
-    type: array
-    items:
-      type: object
-  unique_scans_n:
-    type: integer
-  unknown_ingredients_n:
-    type: integer
-  unknown_nutrients_tags:
-    type: array
-    items:
-      type: object
-  update_key:
-    type: string
-  vitamins_prev_tags:
-    type: array
-    items:
-      type: object
-  vitamins_tags:
-    type: array
-    items:
-      type: object
+  * [Product Base](#cmp--schemas-product-base): Base fields of a product
+  * [Product Misc](#cmp--schemas-product-misc): Miscellaneous but important fields of a product
+  * [Product Tags](#cmp--schemas-product-tags): Tags fields on a product
+  * [Product Nutrition](#cmp--schemas-product-nutrition): Nutrition fields of a product
+  * [Product Ingredients](#cmp--schemas-product-ingredients): Fields about ingredients of a product
+  * [Product Images](#cmp--schemas-product-images): Information about Images of a product
+  * [Product Eco-Score](#cmp--schemas-product-images): Fields related to Eco-Score for a product
+  * [Product Metadata](#cmp--schemas-product-ecoscore): Metadata of a product (author, editors, etc.)
+  * [Product Data Quality](#cmp--schemas-product-quality): fields related to data quality for a product
+  * [Product Knowledge Panels](#cmp--schemas-product-knowledge-panels): Knowledge panels for a product
+
+allOf:
+  - $ref: './product_base.yaml'
+  - $ref: './product_misc.yaml'
+  - $ref: './product_tags.yaml'
+  - $ref: './product_images.yaml'
+  - $ref: './product_ecoscore.yaml'
+  - $ref: './product_ingredients.yaml'
+  - $ref: './product_nutrition.yaml'
+  - $ref: './product_quality.yaml'
+  - $ref: './product_extended.yaml'
+  - $ref: './product_meta.yaml'
+  - $ref: './product_knowledge_panels.yaml'
```

Comment:
The product_hidden.yaml is not there, on purpose ! We want to hide those fields…